pkg/cmd/run_test.go

```diff
@@ -438,6 +438,7 @@ func TestConfigureTraits(t *testing.T) {
 		"--trait", "affinity.pod-affinity=false",
 		"--trait", "environment.container-meta=false",
 		"--trait", "prometheus.pod-monitor=false",
+		"--trait", "telemetry.auto=true",
```

Comment:
@squakez That was the point of this change to test is the run command really creates a trait and not an addon.